pkg/perf/monitor/github.go

```diff
@@ -86,15 +86,13 @@ func (g *Client) ListOpenPRsWithLabel(label string) ([]int, error) {
 }
 
 func prContainsLabel(labels []*github.Label, label string) bool {
+	labelNames := make([]string, 0, len(labels))
 	for _, l := range labels {
-		if l == nil {
-			continue
-		}
-		if l.GetName() == label {
-			return true
+		if l != nil {
+			labelNames = append(labelNames, l.GetName())
 		}
 	}
-	return false
+	return slices.Contains(labelNames, label)
```

Comment:
This change does not make sense. You create a slice with the label names just to check if the label exists. This wastes memory and cpu and make the code more complicated.

We can simply the code using:

```go
func prContainsLabel(labels []*github.Label, label string) bool {
    return slices.ContainsFunc(labels, func(l *github.Label) bool {
        return l.GetName() == label
    })
}
```